### DISCUSSION TITLE
Deposit error

### DISCUSSION THREAD

**Discussion by faizakhalid01** [blockscout/blockscout#9881] [Q&A]
I made a deposit in my trust wallet and transaction is showing in explorer, but not updated in my wallet, screenshots attached, please help
![Screenshot_20240415-161004](https://github.com/blockscout/blockscout/assets/88335636/a5599462-7cbd-4edd-9008-edda91eca077)

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
